### PR TITLE
Remove selectors

### PR DESCRIPTION
**Pull request by vvalentin-lucca**
We remove the configurable selectors of components, mods and states.
(We will see later if the goal is really relevant.)

### CODE REVIEW

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1632

File: packages/scss/src/components/file/component.scss [hunk 1]
```diff
@@ -86,8 +76,8 @@
 		&:hover,
 		&:focus {
 			&:not(:disabled) {
-				~ #{$content} {
-					#{$button} {
+				~ .file-content {
+					.file-button {
 						background-color: var(--palettes-primary-50);
 						color: var(--palettes-primary-500);
 						box-shadow: 0 2px 8px rgba(black, 0.04), 0 1px 2px rgba(black, 0.06), 0 0 0 1px var(--palettes-primary-500) inset;
```
Comment (AlexiaAlarza) [inline packages/scss/src/components/file/component.scss:83]:
```suggestion
						box-shadow: var(--commons-box-shadow-xs), 0 0 0 1px var(--palettes-primary-500) inset;
```

File: packages/scss/src/components/field/states.scss [hunk 2]
```diff
@@ -64,27 +64,27 @@
...
 }
 
-@mixin state($state, $label: '.field-label', $input: '.field-input', $suffix: '.field-suffix') {
+@mixin state {
 	&:not(:disabled, .is-disabled) {
 		background-color: var(--palettes-#{$state}-50);
 
-		~ #{$label},
-		~ #{$suffix} {
+		.field-label,
```
Comment (AlexiaAlarza) [inline packages/scss/src/components/field/states.scss:86]: tu as supprimé le '~' ?

File: packages/scss/src/components/card/component.scss [hunk 3]
```diff
@@ -25,7 +25,7 @@
 			min-height: 3rem;
 		}
 
-		#{$right} {
+		.card-footer {
```
Comment (AlexiaAlarza) [inline packages/scss/src/components/card/component.scss:28]:
```suggestion
		.card-footer-right {
```

File: packages/scss/src/components/radio/states.scss [hunk 4]
```diff
@@ -1,5 +1,5 @@
-@mixin disabled($label: '.radio-label') {
-	~ #{$label} {
+@mixin disabled {
+	.radio-label {
```
Comment (AlexiaAlarza) [inline packages/scss/src/components/radio/states.scss:2]: le '~' manque

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1632

File: packages/scss/src/components/pagination/component.scss [hunk 5]
```diff
@@ -14,19 +14,19 @@
 	}
```
Comment (BertrandPodevin) [inline packages/scss/src/components/pagination/component.scss:15]: You can delete the parameters of the parent `component` mixin
```
component(
	$count: '.pagination-count',
	$navigation: '.pagination-navigation',
	$scrolling: '.pagination-scrolling',
	$separator: '.pagination-count-separator',
	$item: '.pagination-navigation-item'
)
```

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1632

**Comment by LuccaIntegration** [comment]
:woman_cook: http://lucca-front.lucca.local/PR-1632

**Review by BertrandPodevin — APPROVED** [review]

**Pull request merged**